package/CHANGELOG

```diff
@@ -19,6 +19,7 @@ The rules for this file:
   * 0.18.1
 
 Enhancements
+  * Added a box centering trajectory transformation
```

Comment:
Refer to the PR. At the end of the day, it may be worth grouping all the transformations in a single changelog entry.